src/Detail/Blitline/Client/ServiceDescription/Blitline.php

```diff
@@ -31,9 +31,9 @@
             'documentationUrl' => 'http://www.blitline.com/docs/api',
             'parameters'       => array(
                 'src' => array(
-                    'description' => 'The location of the image you wish to process',
+                    'description' => 'The S3 location of the image you wish to process',
```

Comment:
yes is undocumented, only an example is given.

Blitline Example Jobs (http://docs.blitline.com/articles/examples.html)
- Using the permissions you’ve given Blitline for sources:	By default, Blitline assumes a source is public. If you instead want to have Blitline read OUT of your private bucket, you will need to change how the “src” is described. This is an example of using an “s3” source bucket and key

https://www.blitline.com/v3/home/gist?gist_id=3492389